src/Serializer/Filter/GroupFilter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Serializer\Filter;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Group filter.
+ *
+ * @author Baptiste Meyer <baptiste@les-tilleuls.coop>
+ */
+class GroupFilter implements FilterInterface
```

Comment:
So @dunglas, should we keep consistency with ORM filters or should I follow your requested changes?